nipy/modalities/fmri/design_matrix.py

```diff
@@ -61,30 +61,42 @@ def _poly_drift(order, frametimes):
     return pol
 
 
-def _cosine_drift(hfcut, frametimes):
-    """Create a cosine drift matrix
+def _cosine_drift(period_cut, tim):
```

Comment:
In this specific case I tried to keep short names for variable to more easily match the formula to the code, but ok  to change to longer names if people find it easier to read